package.json

```diff
@@ -17,30 +17,33 @@
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "@mui/material": "^5.11.10",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "^6.8.1"
-  },
-  "devDependencies": {
-    "@types/react": "^18.0.27",
-    "@types/react-dom": "^18.0.10",
+    "@types/babel-traverse": "^6.25.7",
+    "@types/compose-function": "^0.0.30",
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
     "@typescript-eslint/eslint-plugin": "^5.54.0",
     "@typescript-eslint/parser": "^5.54.0",
     "@vitejs/plugin-react": "^3.1.0",
+    "babel-traverse": "^6.26.0",
+    "compose-function": "^3.0.3",
     "dependency-cruiser": "^12.10.0",
-    "eslint": "^8.35.0",
-    "effector": "^22.5.1",
+    "effector": "^22.5.2",
+    "effector-logger": "^0.13.7",
```

Comment:
this is dev dependency

Reply:
i`ve been trying to resolve some lint troubles since i deleted package-lock.json . first time working with yarn installer